components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/fragments/policy-add/policy-add.hbs

```diff
@@ -140,8 +146,104 @@
             </select>
         </div>
     </div>
-</form>
+    {{#if @queryParams.subscription}}
+    <div class="page-header">
+        <h2><br></h2>
+        <p class="lead">{{i18n "burst.control.rate.limiting"}}</p>
+    </div>
+
+    <div class="form-group">
+        <label class="control-label col-sm-3">{{i18n "Request.Count"}} </label>
+        <div class="col-sm-6">
```

Comment:
You have only added "col-sm-6" class. add relative classes to col-lg, col-md and col-xs. Otherwise it will be a responsive issues. Check the whole PR. You have multiple places to add this.